fe1-web/src/features/rollCall/components/AttendeeList.tsx

```diff
@@ -0,0 +1,54 @@
+import { ListItem } from '@rneui/themed';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View } from 'react-native';
+
+import { PoPIcon } from 'core/components';
+import { PublicKey } from 'core/objects';
+import { Color, Icon, List, Typography } from 'core/styles';
+
+const AttendeeList = ({ popTokens }: IPropTypes) => {
```

Comment:
This should probably present an alternative content if there are no poptokens to list .

Reply:
what do you think of? some text?

Reply:
Yup, some text of some kind 😅  I'd have to think longer about the content.